provisioners/configure.sh

```diff
@@ -22,14 +22,8 @@ echo newrelic-php5 newrelic-php5/application-name string $NEW_RELIC_APPLICATION_
 echo newrelic-php5 newrelic-php5/license-key string $NEW_RELIC_LICENSE_KEY | debconf-set-selections
 
 echo "Add custom sources"
-# Add mysql key
-sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 5072E1F5
-# Add New Relic key
-curl -s https://download.newrelic.com/548C16BF.gpg | apt-key add -
-# Add node key
-curl -s https://deb.nodesource.com/gpgkey/nodesource.gpg.key | apt-key add -
-# Add custom sources
-cp $TEMPLATES_PATH/etc/apt/sources.list.d/* /etc/apt/sources.list.d/
+cp $TEMPLATES_PATH/usr/share/keyrings/*.gpg /usr/share/keyrings/
+cp $TEMPLATES_PATH/etc/apt/sources.list.d/*.list /etc/apt/sources.list.d/
```

Comment:
I believe this should be:

```
cp $TEMPLATES_PATH/etc/apt/sources.list.d/*.sources /etc/apt/sources.list.d/
```

(`.*list` => `.*sources`)

Reply:
Oops, yes, you're absolutely right!